src/__tests__/commands/utilities/server.test.ts

```
@@ -0,0 +1,419 @@
+			mockGuild.iconURL = vi.fn().mockReturnValue(undefined);
+
+			await serverExecute(mockInteraction as ChatInputCommandInteraction);
+
+			const replyCall = vi.mocked(mockInteraction.reply).mock.calls[0];
+			const embed = replyCall[0].embeds![0];
+			expect(embed.data.thumbnail).toBeUndefined();
+		});
+	});
+
+	describe('Error Scenarios', () => {
+it('should propagate reply failure', async () => {
```

ESLint

[error] Expected indentation of 2 tabs but found 0.

(indent)